.functions.php

```diff
@@ -77,8 +77,8 @@ function updateSession($sessionID, $connection){
 
 function openDatabaseConnection(){
   try{
-    require_once '.db_config.php';
```

Comment:
This might work better is a `require_once` because that function produces a fatal error if the file can't be found, whereas `include` only produces a warning.

Reply:
It should produce a fatal error it can't get to the database. The reason it can't be a require_once is if we do something like `require_once('.db_config.php');` in an external file and then we `require_once('.functions.php')` when it goes to require the db_config.php here it won't based on the way require_once works leading to a database connection error as the credentials are undefined.